app/queries/coach_stats_resolver.rb

```diff
@@ -0,0 +1,32 @@
+class CoachStatsResolver < ApplicationQuery
+  property :coach_id
+  property :course_id
+
+  def coach_stats
```

Comment:
@harigopal: As the stats are shown in the coach-info from for a course, should it be scoped course wise?

Reply:
Fixed in 2e02701aab71667c4604e351983484648d1bd8ea.